components/layout/index.css

```diff
@@ -1,19 +1,20 @@
 .layout {
-  height: 100%;
+  min-height: 100vh;
```

Comment:
It should expand automatically but if not, okay. I will take a look at it later.

Reply:
Unfortunately, it didn't work. Thanks for looking at it later. :)